intellij-ext/src/main/kotlin/com/github/metalbear/intellijplugin/MirrordDialogBuilder.kt

```diff
@@ -0,0 +1,59 @@
+package com.github.metalbear.intellijplugin
+
+import com.intellij.openapi.ui.DialogBuilder
+import com.intellij.ui.components.JBList
+import java.awt.BorderLayout
+import java.awt.GridLayout
+import javax.swing.JCheckBox
+import javax.swing.JLabel
+import javax.swing.JPanel
+import javax.swing.JSeparator
+import javax.swing.border.EmptyBorder
+
+
+class MirrordDialogBuilder {
+    private val heading: String = "mirrord"
+    private val labelName: String = "Select pod to impersonate"
+
+    fun createMirrordKubeDialog(pods: JBList<String>, fileOpsCheckbox: JCheckBox, remoteDnsCheckbox: JCheckBox): JPanel {
+        val dialogPanel = JPanel(BorderLayout())
+        val label = JLabel(labelName)
+
+        val podPanel = JPanel(GridLayout(2, 1, 10, 5))
+        podPanel.add(label, BorderLayout.NORTH)
+        podPanel.add(pods)
+
+        dialogPanel.add(podPanel, BorderLayout.WEST)
+
+        dialogPanel.add(JSeparator(JSeparator.VERTICAL),
+                BorderLayout.CENTER);
+
+        val checkBoxPanel = JPanel(GridLayout(2, 1, 10, 2))
+        checkBoxPanel.add(fileOpsCheckbox)
+        checkBoxPanel.add(remoteDnsCheckbox)
+
+        dialogPanel.add(checkBoxPanel, BorderLayout.EAST)
+
+        return dialogPanel
+    }
+
+    fun createMirrordNamespaceDialog(namespaces: JBList<String>): JPanel {
+        val dialogPanel = JPanel(BorderLayout())
+        val label = JLabel("Select Namespace to use")
+        val border = EmptyBorder(5, 20, 5, 20)
```

Comment:
Why not initialize straight into label.border?

Reply:
iirc setter returns a unit, not this so cant do that here or do you mean something else?

Reply:
I meant why not just do `label.border = EmptyBorder(5, 20, 5, 20)`?